charts/gardener-extension-shoot-cert-service/templates/rbac.yaml

```diff
@@ -67,12 +67,8 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - "configmaps"
   - "secrets"
   - "events"
-  - "services"
-  - "pods"
-  - "serviceaccounts"
```

Comment:
Don't we need permissions for `services` and `serviceaccounts` because we deploy https://github.com/gardener/gardener-extension-shoot-cert-service/blob/master/charts/internal/shoot-cert-management-seed/templates/service.yaml and https://github.com/gardener/gardener-extension-shoot-cert-service/blob/master/charts/internal/shoot-cert-management-seed/templates/serviceaccount.yaml?

Reply:
These objects are deployed by the gardener-resource-manager. The extension only creates managed resource and secret, right?

Reply:
Right, thanks for reminding 👍